tests/testYearWrap.py

```diff
@@ -26,13 +26,19 @@
 import unittest
 import lsst.utils.tests
 import lsst.ctrl.stats.records as recordslib
-from datetime import date
+import datetime
+from dateutil import tz, parser
 from lsst.ctrl.stats.reader import Reader
 
 
 def setup_module(module):
     lsst.utils.tests.init()
 
+def tolocal(dateVal):
```

Comment:
is this a function that would be useful to the general user or only useful to these tests? If it's the latter then making a `helper.py` in the `tests/` directory should be sufficient. `pytest` won't find any tests in there. The more robust way would be to put a `helper/` directory in there and put the code in there.